Bottom: d06309be5046fd59e64d2b94132a27ab595f47b4
Top:    48f2fdc5c3344d9ac0339e94aa1ef4818989e302
Author: Ben Greear <greearb@candelatech.com>
Date:   2013-02-06 09:08:30 -0800

macvlan: Enable qdisc backoff logic.

This allows macvlans to apply backpressure to calling code
in the transmit path when the underlying qdisc cannot accept
additional packets.

Signed-off-by: Ben Greear <greearb@candelatech.com>


---

diff --git a/include/linux/if_vlan.h b/include/linux/if_vlan.h
index a5f6ce6..91a82b9 100644
--- a/include/linux/if_vlan.h
+++ b/include/linux/if_vlan.h
@@ -102,6 +102,7 @@ struct vlan_pcpu_stats {
 	struct u64_stats_sync	syncp;
 	u32			rx_errors;
 	u32			tx_dropped;
+	u32                     collisions;
 };
 
 #if defined(CONFIG_VLAN_8021Q) || defined(CONFIG_VLAN_8021Q_MODULE)
